polywrap.build.yaml

```diff
@@ -1,10 +1,19 @@
-format: 0.1.0
-docker:
-  name: ens-wrapper-build
 config:
-  node_version: "17.9.1"
+  node_version: 17.9.1
   include:
     - ./package.json
     - src/index.ts
     - src/contracts
     - src/utils.ts
+format: 0.3.0
```

Comment:
why do we have this duplicated? is this needed? would it be possible to remove this build manifest?

Reply:
Nice catch ser, fixed.